BUILD.gn

```diff
@@ -266,7 +266,7 @@ if (is_clang) {
 if (is_linux) {
   # Use system fontconfig. This avoids a startup hang on Ubuntu 16.04.4 (see
   # issue #2424).
-  assert(!use_bundled_fontconfig)
+#  assert(!use_bundled_fontconfig)
```

Comment:
This one is too to be reconsidered later when we can run the thing and see what is going on.